src/remount-fs/remount-fs.c

```diff
@@ -69,11 +70,46 @@ static int do_remount(const char *path, bool force_rw, Hashmap **pids) {
         return track_pid(pids, path, pid);
 }
 
+static int mount_point_matches(const struct mntent *me) {
```

Comment:
Frankly, not too happy with the approach taken here (or the current remount-fs in general as it bypasses service manager).

I'd like to port it over to #36337 in the future, and this should